README.md

```diff
@@ -27,3 +28,15 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+## Invariant Answer
+
+What we can always assume is true is that the first two items in our array are defined as `x[0] = 0` and `x[1] = 1`, or in terms of the Fibonacci sequence $F_1 = 0$ and $F_2 = 1$. There is never a case when this is not true or it would not be the Fibonacci sequence by definition. 
+
+Let's say we are finding `fib(4)`, which should give us `[0, 1, 1, 2, 3]`.
+
+The function passes through `fib` because our iteration is not 0 or 1. Then it goes into `fibFind` and our list is `x = [0,1]`. 
+
+Then the function checks if n == 2, assuming that the list is already defined for $n_0$ and $n_1$. n is not 2 in our case so it recursively calls `fibFind(n-1)` where n is 3, and then does it again until n == 2. Then we finally run `x.push(1)`, or the sum of 0 and 1, I used 1 to save calculating what we can already assume from the invariant when we get to this case. Then we return and get to the recursive case when n = 3 and we run `x.push(x[n-1]+x[n-2])`, the formula for the sequence and build our array and return it when finished. 
+
+So in conclusion, the invariant for this function is that the first two items of our list is defined as `[0,1]` and our code assumes this in runtime. 
```

Comment:
Ok, but how does this help to you to show that a particular iteration will produce the correct result?

Reply:
Because the first two numbers of the Fibonacci sequence is the invariant and therefore doesn't change, we are always using the same two numbers in the same formula to produce the correct result every time. A particular iteration will always start the same using our invariant and will always consistently make the same result over and over again.

Reply:
The same result being the correct result every time because it is based on our invariant, Hopefully I'm not being too wordy.